doc/source/tune/guides/plot_tune-usage.rst

```diff
@@ -0,0 +1,375 @@
+.. _tune-user-guide:
+
+Tune User Guide
+===============
+
+.. warning:: Before you continue, be sure to have read :ref:`tune-60-seconds`.
+
+This document overviews features and different configurations of Tune.
```

Comment:
This document provides an overview of the core concepts as well as some of the configurations for Tune Runs.

Reply:
```suggestion
This document provides an overview of the core concepts as well as some of the configurations for running Tune.
```